release-notes.hbs.md

```diff
@@ -144,6 +144,18 @@ enable TUF:
 
 - SCST - Store returns an expired certificate error message when a CA certificate expires before the app certificate. For more information, see [CA Cert expires](scst-store/troubleshooting.hbs.md#ca-cert-expires).
 
+#### <a id='1-9-0-ssc-ui-ki'></a> v1.9.0 Known issues: Supply Chain UI
+
+- When accessing the supply chain tab in the Tanzu Developer Portal, users might encounter an error related to data.packaging.carvel.dev. This issue arises even for supply chains that do not generate Carvel packages, indicating an unexpected configuration requirement. The error message displayed is related to permission issues and JSON parsing errors, specifically mentioning that the user "system:serviceaccount:tap-gui:tap-gui-viewer" cannot list resource "packages" in the API group "data.packaging.carvel.dev" at the cluster scope. Additionally, an unexpected non-whitespace character is reported after JSON at position 4.
+
+Workaround: A temporary solution involves applying an RBAC configuration that includes permissions (get, watch, list) for the resources within the data.packaging.carvel.dev API group. This configuration mitigates the issue but it is highlighted that such a requirement should not be mandated for supply chains not generating Carvel packages.
+
+The error goes away after below rbac config is applied:
```

Comment:
**SUGGESTION**

Suggest rephrasing to use more active verb: "Configuring RBAC to allow access to the Carvel package resource eliminates the error message:"

Also suggest adding the GVK section to this snippet, to more clearly indicate where this change needs to happen (i.e. in Kubernetes, not in Backstage):

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
- apiGroups: [data.packaging.carvel.dev]
  resources: [packages]
  verbs: ['get', 'watch', 'list']
- … 
```

Reply:
Done!